spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/IntegerEncoderDecoder.java

```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```

Comment:
Hmmm, turns out the license plugin I used does not add a new line, even if I put it there. What a shame. gonna do some `sed`ding